composer.json

```diff
@@ -22,7 +22,7 @@
         "zendframework/zend-diactoros": "^1.1",
         "zendframework/zend-expressive-router": "^1.1",
         "zendframework/zend-expressive-template": "^1.0.1",
-        "zendframework/zend-stratigility": "^1.1"
+        "zendframework/zend-stratigility": "^1.2.0"
```

Comment:
They're semantically equivalent. `^` will allow from that version to next
major. (vs `~`, which limits to the same minor version.)
On Mar 21, 2016 6:23 PM, "Michaël Gallego" <notifications@github.com> wrote:

> In composer.json
> <https://github.com/zendframework/zend-expressive/pull/309#discussion_r56914722>
> :
>
> > @@ -22,7 +22,7 @@
> >          "zendframework/zend-diactoros": "^1.1",
> >          "zendframework/zend-expressive-router": "^1.1",
> >          "zendframework/zend-expressive-template": "^1.0.1",
> > -        "zendframework/zend-stratigility": "^1.1"
> > +        "zendframework/zend-stratigility": "^1.2.0"
>
> ^1.2 instead?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/zendframework/zend-expressive/pull/309/files/889ed8b8ddeb9c92de319184aa9377d22d9112f5#r56914722>
>